provisioning/roles/mariadb/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+- set_fact:
+    database_type: mysql
+
+- name: Add MariaDB apt key
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: F1656F24C74CD1D8
+  become: yes
+
+- name: Add MariaDB repo
+  apt_repository:
+    repo: "deb http://mirror.mva-n.net/mariadb/repo/{{ mariadb_version }}/debian stretch main"
```

Comment:
This shouldn't be hardcoded to Debian Stretch. If we don't want to support earlier version that's possible but we still need Ubuntu compatibility.

Reply:
Indeed. @SergioMendolia please either support other versions/distributions (you can check the PHP roles for examples on how to do that), or explicitely fail if the OS version/distribution is not stretch and document that the role is only available for debian stretch.